src/Symfony/Bundle/FrameworkBundle/Resources/config/feature_flag_routing.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+
+        ->set('feature_flag.routing_expression_language_function.is_enabled', \Closure::class)
+            ->factory([\Closure::class, 'fromCallable'])
+            ->args([
+                [service('feature_flag.feature_checker'), 'isEnabled'],
+            ])
+            ->tag('routing.expression_language_function', ['function' => 'feature_is_enabled'])
+
+        ->set('feature_flag.routing_expression_language_function.get_value', \Closure::class)
+            ->factory([\Closure::class, 'fromCallable'])
+            ->args([
+                [service('feature_flag.feature_checker'), 'getValue'],
```

Comment:
`getValue`seems a bit too generic for routing expression. maybe something more specific ?

Reply:
Are you talking about comparing the feature value to an expected value? If so, the `isEnabled` method can be used instead, making `getValue` useless.

Reply:
my bad I misread the purpose of `getValue()` closing this thread.